### DISCUSSION TITLE
Share a global state between files in solara

### DISCUSSION THREAD

**Discussion by rururu** [widgetti/solara#618] [Q&A]
Hi,
Please, share an example of how to share a global state between files in solara.
Thanks in advance.
Sincerely,
  Ru.

**Comment by iisakkirotko** [comment]
Hi @rururu!

Thanks for the suggestion! We'll take a look at adding an example of this. In the meantime, [this thread](https://www.answeroverflow.com/m/1233330090021093426) should give you some pointers.

**Reply by rururu** [reply]
Thank you,  @iisakkirotko! This was helpful.
